src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -231,7 +232,6 @@ private void calculateSensitivityValuesForAContingency(DcLoadFlowContext loadFlo
             DenseMatrix postContingencyFlowStates = engine.run(newFlowStates);
             DenseMatrix postContingencyFactorStates = engine.run(newFactorStates);
             calculateSensitivityValues(factors, postContingencyFactorStates, postContingencyFlowStates, contingency, resultWriter, disabledNetwork);
```

Comment:
Since nexFlowStates and newFactorSates are modified by run, and since they can be the input values factorSate and flowState the javadoc of calculateSensitivityValuesForAContingency should explicit that the matrices may be modified.